lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -140,12 +140,12 @@ def store_attributes
 
       def descendants(filters = {})
         assert_valid_filters!(filters)
-        Assignment.descendants_of(self).where(filters)
+        Assignment.descendants_of(self.id).where(filters)
```

Comment:
To remove a db call since we're not yet using AR chaining. But no worries –  I changed it back.